tripleblindmarket/covid-safe-paths
# Fix incorrect "plural" keys, add key checks for new PRs

- Cleans up some incorrect plural keys.
- Moves some keys to `share.*` and `import.google.*`
- Add a check for missing keys. [Example failure](https://github.com/tripleblindmarket/covid-safe-paths/pull/648/checks?check_run_id=612872599)

## How to test

Check that the strings show in the app. Check that the renamed keys are matched with old values in Lokalise
